rootfs/etc/profile.d/dietpi-unsupported_terminal.sh

```diff
@@ -28,8 +28,7 @@ if [[ $SSH_TTY ]] && ! toe -a | grep -q "$TERM"; then
 	G_WHIP_DEFAULT_ITEM=1
 	
 	(( ! G_PROGRAM_NAME='Unsupported SSH client terminal' G_WHIP_MENU "[WARNING] The terminal passed by your SSH client (TERM=$TERM_old) is not supported by this DietPi server.\n
-To enable usage, we were fooling the server by setting \"TERM=$TERM\". This is not the cleanest solution, since commands might expect colours or formats, that are not supported by the actual terminal.\n
-Please change your SSH clients terminal (passed $TERM string)$ncurses_term." && G_WHIP_RETURNED_VALUE )) && G_AGI ncurses-term
+Please change your SSH clients terminal (passed $TERM string)$ncurses_term." && (( $G_WHIP_RETURNED_VALUE )) && G_AGI ncurses-term ))
```

Comment:
Ah sorry, you added `))` after `G_AGI ncurses-term` as well? Should be removed as well.

So `G_WHIP_MENU` main command, outside of arithmetic,
if it succeeds, then check it's output value `$G_WHIP_RETURNED_VALUE` inside arithmetic (as result can only be 0 or 1),
if menu entry 1 was chosen, then install ncurses-term 🙂.